chain/network/src/types.rs

```diff
@@ -753,21 +753,17 @@ pub struct NetworkConfig {
 
 impl NetworkConfig {
     pub fn verify(&self) {
-        let mut some_error = false;
-
         if self.ideal_connections_lo + 1 >= self.ideal_connections_hi {
             error!(target: "network",
             "Invalid IDEAL_CONNECTIONS values. LO({}) > HI({})",
             self.ideal_connections_lo, self.ideal_connections_hi);
-            some_error = true;
         }
 
         if self.ideal_connections_hi >= self.max_peer {
             error!(target: "network",
-                "Inestable IDEAL_CONNECTIONS_HI({}) compared with MAX_PEERS({})",
+                "Inestable IDEAL_CONNECTIONS_HI({}) compared with MAX_PEER({})",
```

Comment:
What is `inestable`?

Reply:
Maybe inestable is not the best word, but if hi >= max peer then it is possible that the node is at Max capacity and don't let other nodes connect to it. It will not remove any connections.

Reply:
Why is it "max peer" now? that doesn't make sense. 
If you want to rename, it should MAX_NUM_PEERS and IDEAL_NUM_CONNECTIONS_HI. 
Also should it be lowercase?
Error message can be "MAX_NUM_PEERS is below IDEAL_NUM_CONNECTIONS_HI, which may lead to connection saturation and declining new connections." or something like this.